projects/microphysics/train/run_arch.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+config_file=$1
+model_type=$2
+model_prefix=$3
+
+model_name=${model_prefix}-${model_type}
+out_url=$(artifacts resolve-url vcm-ml-experiments microphysics-emulation ${model_name})
```

Comment:
Thx for updating this.  Removes some of the previous redundancy in the blob naming